stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/selectedmessage/SelectedMessageDialog.kt

```diff
@@ -57,7 +55,7 @@ internal fun SelectedMessageDialog(
             shape = shape,
             backgroundColor = ChatTheme.colors.barsBackground
         ) {
-            Column(modifier = Modifier.padding(top = 12.dp)) {
+            Column(modifier) {
```

Comment:
Removing padding from the containers containing slots and moving it to inner slot components.